GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -951,7 +1060,8 @@ private String sanitize(String s) {
                                         + " - " + level;
                                 createCell(row, offset++, levelName, headerStyle);
                             }
-                        } else {
+                        }
+                        else {
```

Comment:
Yes, but the problem is that the first part needs to be done for all columns, including for the first column of an option question. The check for the option question comes after that, and if valid, creates the additional option columns.